config/core/manager_arguments.yaml

```diff
@@ -1,4 +1,4 @@
 - op: replace
-  path: "/spec/template/spec/containers/0/args"
```

Comment:
We can remove this file because the manager_auth_proxy_patch.yaml you're adding here handles this. You can also remove the file of the same name from config/default, and get rid of `patchesJson6902` in its kustomization.yaml (right @chuckha?)